transforms/universal/pdf2md/python/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM docker.io/python:3.11.9-slim-bookworm
```

Comment:
Why the change from FROM docker.io/python:3.10.14-slim-bullseye

Reply:
we have python 3.11 as minimum version required